dashboard/test/helpers/levels_helper_test.rb

```diff
@@ -476,12 +476,12 @@ def stub_country(code)
     script_data, _ = ScriptDSL.parse(input_dsl, 'test_level_links')
 
     script = Script.add_script(
-      {name: 'test_level_links'},
+      {name: 'test level links'},
```

Comment:
can you change the script name to use dashes instead of spaces, here and in the previous test case? I'm afraid this will give the wrong idea otherwise, because script names cannot include spaces or underscores. this is enforced here: https://github.com/code-dot-org/code-dot-org/blob/d5cfdb590d87ce660a952ddf6b1bcd7ff22bf7bd/dashboard/app/models/concerns/serialized_to_file_validation.rb#L13